bundled/tool/install_telemetry_dependencies.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+import sys
+from pathlib import Path
+
+def install_dependencies(extension_root_dir):
+    """
+    Install dependencies required for the Kedro extension.
+
+    Args:
+        extension_root_dir (str): The root directory of the extension.
+    Raises:
+        ImportError: If the required dependencies are not found.
+    """
+    ...
+    libs_path = Path(extension_root_dir) / "bundled" / "libs"
+    requirements_path = Path(extension_root_dir) / "kedro-viz-requirements.txt"
+
+    try:
+        import kedro_telemetry
+        from packaging.version import parse
+
+        version = parse(kedro_telemetry.__version__)
+        if version.major<1 and version.minor<6: # at least >0.6.0
+            raise ImportError("kedro-telemetry version must be >=0.6.0")
```

Comment:
Do we need to do version checks for kedro-viz dependencies as well ?

Reply:
@jitu5 I think so, can you address this separately in other PR?

Reply:
@noklam  https://github.com/kedro-org/vscode-kedro/pull/96